src/plus/integrations/authentication/gitlab.ts

```diff
@@ -84,3 +83,19 @@ export class GitLabAuthenticationProvider extends LocalIntegrationAuthentication
 		};
 	}
 }
+
+export class LocalGitLabAuthenticationProvider extends GitLabSelfHostedAuthenticationProvider {
```

Comment:
There could be future side effects with this extending the self hosted provider. Would it work to do this instead?

```suggestion
export class GitLabLocalAuthenticationProvider extends LocalIntegrationAuthenticationProvider<GitLabId> {
```

Reply:
Fixed